ttnn/cpp/ttnn/operations/experimental/ccl/all_gather_matmul/device/all_gather_matmul_op.cpp

```diff
@@ -73,7 +73,7 @@ std::vector<ttnn::SimpleShape> AllGatherMatmul::compute_output_shapes(const std:
 
     // Matmul shape
     ttnn::SimpleShape matmul_output_shapes =
-        this->matmul_struct.compute_output_shapes({input_tensors[1], input_tensors[2]})[0];
+        this->matmul_struct.compute_output_specs({input_tensors[1], input_tensors[2]}, {})[0].logical_shape();
```

Comment:
Not a blocker, but should also switch over to use `compute_output_specs` for all gather matmul

Reply:
issue created at https://github.com/tenstorrent/tt-metal/issues/15997